build-infrastructure/codebuild-devbuild-stack.yml

```diff
@@ -0,0 +1,235 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: A Cloudformation template to build Agent artifacts on PR creation and modification. It spawns CodeBuild projects for different architectures which trigger agent artifact builds for PR creation and modification, and store the artifacts in an S3 bucket.
+
+Parameters:
+  GithubFullRepoName:
+    Type: String
+    Description: Full name of the agent repository to use (e.g. https://github.com/aws/amazon-ecs-agent.git)
+  GithubBranchName:
+    Type: String
+    Description: Name of the base branch to use to build, PRs to which will trigger builds (e.g. refs/heads/dev)
+  BuildBucketName:
+    Type: String
+    Description: Name of the S3 bucket which will store artifacts
+  BuildBucketArn:
+    Type: String
+    Description: ARN of the S3 bucket which will store artifacts
+  BuildProjectName:
+    Type: String
+    Description: Base name of CodeBuild projects - will start different CodeBuild projects with different suffixes (-amd, -arm) for different artifacts
+
+Resources:
+  ArmProject:
+    Type: 'AWS::CodeBuild::Project'
+    Properties:
+      Artifacts:
+        Location: !Ref BuildBucketName
+        NamespaceType: NONE
+        OverrideArtifactName: true
+        Packaging: NONE
+        Path: development
+        Type: S3
+      BadgeEnabled: false
+      ConcurrentBuildLimit: 10
+      Description: A sample agent build on pr
```

Comment:
we'll want to update this description once we're ready to synthesize this for production.

Reply:
Yeah, i missed updating this one, will do that